README.md

```diff
@@ -1,6 +1,7 @@
 # Uploadcare picture
 
-Uploadcare responsive `<picture>` component using [Uploadcare CDN API](https://uploadcare.com/docs/delivery/file_api/#processing)
+This is a component that makes your `<picture>` responsive using
```

Comment:
Component didn't make some picture responsive, component is a responsive picture